### ISSUE TITLE
Block multiple components with BlockUI

### ISSUE THREAD

**Issue by cagataycivici** [primefaces/primefaces#50]
t would be helpful if we had possibility to to add more than one area to be blocked on particular ajax action

``` xml
<p:blockUI block="area1, area2" target"someTarget"/>
```

**Comment by sudheerj** [comment]
Duplicate of issue https://github.com/primefaces/primefaces/issues/717

**Comment by sudheerj** [comment]
Already fixed for 5.3.1,5.2.13 and 6.0 releases.Now we can use PFS or comma separated mutiple component ids.
